scripts/managed-services/get-kafka-instance-state/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+file: get-kafka-instance-state.sh
+name: managed-services/get-kafka-instance-state
+description: |
+  Print out the logs for the specified Kafka instance.
+  The following commands will create a directory with all instance information:
+
+  ocm backplane managedjob logs <job-name> > mykafkadebug.tgz
+  tar -xvf mykafkadebug.tgz
+author: racheljpg
+allowedGroups:
+  - SREP
+  - CSSRE
+  - CEE
+rbac:
+  roles:
+    - namespace: "redhat-kas-fleetshard-operator"
+      rules:
+        - verbs:
+            - "get"
+            - "list"
+          apiGroups:
+            - ""
+          resources:
+            - "pods/exec"
+        - verbs:
+            - "create"
+          apiGroups:
+            - ""
+          resources:
+            - "pods/exec"
+      clusterRoleRules:
+        - verbs:
```

Comment:
The script needs to `oc exec <pod>` from any kafka instance namespace.  We don't know the names of the namespaces as they will be defined in advance.  I would have thought we'd have need cluster role permission.